cms/templates/js/course-outline.underscore

```diff
@@ -246,6 +246,48 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <% if (course.get('self_paced') && course.get('is_custom_pls_active') && xblockInfo.get('due_num_weeks')) { %>
+                    <div class="status-grading">
+                        <p>
+                            <span class="icon fa fa-calendar" aria-hidden="true"></span>
+                            <span class="status-custom-grading-date">
+                                <%- edx.StringUtils.interpolate(
+                                        ngettext('Custom due date: {dueNumWeeks} week from enrollment',
+                                        'Custom due date: {dueNumWeeks} weeks from enrollment',
+                                        xblockInfo.get('due_num_weeks')),
+                                        {
+                                            dueNumWeeks: xblockInfo.get('due_num_weeks')
+                                        }
+                                    )
+                                %>
+                            </span>
+                        <p>
+                    </div>
+                <% } %>
+            <% } else if (course.get('self_paced') && course.get('is_custom_pls_active') && xblockInfo.get('due_num_weeks')) { %>
+                <div class="status-grading">
+                    <p>
+                        <span class="sr status-grading-label"> <%- gettext('Graded as:') %> </span>
+                        <span class="icon fa fa-check" aria-hidden="true"></span>
+                        <span class="status-grading-value"> <%- gradingType %> </span>
+                    </p>
+                </div>
+                <div class="status-grading">
+                    <p>
+                        <span class="icon fa fa-calendar" aria-hidden="true"></span>
+                        <span class="status-custom-grading-date">
+                            <%- edx.StringUtils.interpolate(
+                                    ngettext('Custom due date: {dueNumWeeks} week from enrollment',
```

Comment:
Was this the text UX/Product came up with? I feel like it is odd to call out the due date as having been "Custom". They are all always custom. Feels more tied up into our thinking of them as custom as opposed to the default PLS dates, but apply less from the user perspective.

Reply:
yup it's text UX/Product came up with